_docs/coc.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+category: Logistics
+title: Code of Conduct
+order: 1
+permalink: /code_of_conduct.html
+---
+
+# Code of Conduct
+
+UWB Hacks the Cloud is going to be amazing! Let's lay some ground rules to make sure everyone has the best experience possible.
+
+## TL;DR
+
+The gist of the Code of Conduct is:
+
+* be respectful to each other
+* don't plagarize other people's work
+* breaking these rules will have consequences
+
+### Harassment
+
+Our hackathon is dedicated to providing a safe and comfortable environment and harassment-free experience for everyone, regardless of the following:
+
+* gender
+* gender identity and expression
+* age
+* sexual orientation
+* disability
+* physical appearance
+* body size
+* race
+* ethnicity
+* nationality
+* religion
+* political views
+* previous hackathon attendance or lack of
+* computing experience or lack of
+* chosen programming language or tech stack
+
+We do not tolerate harassment of hackathon participants in any form. Sexual language and imagery is not appropriate at any hackathon venue; this includes the following:
+
+* hacks
+* talks, presentations, or demos
+* workshops
+* any parties associated to the hackathon
+* social media
+* any other online media
+
+### Plagarism
+
+We do not tolerate intentional direct plagarism of code or visual content from other participants or readily available resources on the internet.
+
+If your project uses visual or programmatic elements that your team didn't create from scratch, you are expected to attribute credit to the original author(s).
+
+Please use common sense, and take this opportunity to showcase your own individual creativity.
+
+### Consequences
+
+Hackathon participants violating these rules may be sanctioned at the discretion of the hackathon organisers. Any of the following consequences may result:
+
+* being asked to leave the event
+* being banned from future events hosted by the organizers
+* having awards revoked
+
+### Inquiries
+
+If you have any questions about these rules, please contact [the hackathon organizers](https://uwbacm.com).
```

Comment:
A few of the code of conduct examples I just scoped out for inspiration have a way for people to report violations of the code of conduct anonymously which I thought might make it easier for people to speak up if they're feeling uncomfortable, but also might pose a small logistical challenge with organizers having to monitor the form submissions during the hackathon 🤷‍♀  If we want to implement something like that, I can write up a Google Form people can use as a reporting tool and add it here.
```suggestion
If you have any questions about these rules, please contact [the hackathon organizers](https://uwbacm.com).

If you feel uncomfortable or think there may be a potential violation of the code of conduct, please report it immediately using the [Code of Conduct Violation Report]() form. All reporters have the right to remain anonymous.
```